apiv2/database/migrations/versions/20250219_050850_autogenerated.py

```diff
@@ -26,7 +26,8 @@ def upgrade() -> None:
 def downgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column(
-        "frame", sa.Column("raw_angle", sa.DOUBLE_PRECISION(precision=53), autoincrement=False, nullable=True),
+        "frame",
+        sa.Column("raw_angle", sa.DOUBLE_PRECISION(precision=53), autoincrement=False, nullable=True),
```

Comment:
Parking the comment here as it is related to migrations. It looks like we are missing the migrations related to table creation for the `PerSectionParameter`.

Reply:
Added it in. I forgot to stage it.